### PR TITLE
Added my blog post

### PR DESCRIPTION
**Pull request by DotSlashTX**
Author : Prasenjit Gautam

Blog on my experince during GCI

### CODE REVIEW

File: _data/blogs.yml [hunk 1]
```diff
@@ -42,3 +42,8 @@
   title: Interview with Prasenjit Gautam, GCI Contestant and Open Source Contributor
   img: 7.jpg
   author_name: Kuba Michalski  
+  
+- url: http://https://codegutter.wordpress.com/2017/12/09/root-to-open-source-with-fossasia/
```
Comment (harshit98) [inline _data/blogs.yml]: Correct the address link. `http` is being repeated two times.

Comment (akshatnitd) [inline _data/blogs.yml]: The url is invalid. Kindly correct it :)

**Review by akshatnitd — CHANGES_REQUESTED** [review]
Make the changes :)

**Review by anshumanv — CHANGES_REQUESTED** [review]
Please add the image with this PR.

**Pull request closed without merging**